LondonBritishLibrary/orient/BLorient782.xml

```diff
@@ -0,0 +1,507 @@
+                                    <dimensions type="outer" unit="in">
+                                        <height>12.5</height>
+                                        <width>10.75</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>One unnumbered blank leaf at the end.</foliation>
+
+                            </supportDesc>
+
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="34 35"/>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Written in small, elegant character (<locus from="1r"></locus>).
+                                </desc>
+                                <date notBefore="1563" notAfter="1597"/>
+                                <persName ref="PRS13110TaklaM" role="scribe"></persName>
+                            </handNote>
+                            <handNote script="Ethiopic" xml:id="h2">
+                                <desc>Written in small, elegant character (<locus from="65r"></locus>).
+                                </desc>
+                                <date notBefore="1563" notAfter="1597"/>
+                                <persName ref="PRS13111TawaldaM" role="scribe"></persName>
+                            </handNote>
+                            <handNote script="Ethiopic" xml:id="h3">
+                                <desc>Written in small, elegant character (<locus from="129r"></locus>).
+                                </desc>
+                                <date notBefore="1563" notAfter="1597"/>
+                                <persName ref="PRS13112AmdaM" role="scribe"></persName>
+                            </handNote>
+
+                        </handDesc>
+                        <additions>
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1r"></locus>
+                              <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                  <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                              <q xml:lang="gez">ተልሚድ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                        </q>
+                            </item>
+                          </list>
+                          <list>
+                            <item xml:id="e1">
+                              <desc>The names of three scribes are mentioned throught the manuscript:
+                              ዘ<persName ref="PRS13110TaklaM" role="scribe">ተክለ፡ ማርያም፡</persName>
+                              (<locus from="1r" to="57r"></locus>),
```

Comment:
I think it would be good to add the complete locus in the description of the hands as well

Reply:
This is not easy, there are folia between the last mention of the previous scribe and the first mention of the next scribe which can not be attributed with certainty...

Reply:
Ah, I see